design-editor/src/panel/preview/preview-element.js

```diff
@@ -141,6 +141,21 @@ class Preview extends DressElement {
             contentDoc.body.dispatchEvent(event);
         }
     }
+
+    /**
+     * Add temporary document for preview content
+     * @param {string} contents whole HTML code of currently edited document
+     * @param {string} location path to directory where 
+     * currently edited HTML file exists
+     * @returns {string} absolute path to temporary file
+     */
+    createPreviewDocument(contents, location) {
+        const relativePathToFile = pathUtils.joinPaths(location, 'temporary-preview.html');
+        this.fsPath = pathUtils.createProjectPath(relativePathToFile);
+        fs.writeFile(this.fsPath, contents, (err) => {
+            if(err) throw err});
```

Comment:
add space after `if`, semicolon after `err` and `});` to the new line :p

Reply:
it would probably make more sense to return path after being sure that successfully created and written to a file. 
In this case, we can return a promise returning `pathUtils.createProjectPath(relativePathToFile, true);`